modules/monitoring-support-considerations.adoc

```diff
@@ -0,0 +1,24 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// * assemblies/configuring-the-monitoring-stack.adoc
+
+[id="support-considerations_{context}"]
+= Support considerations for monitoring
+
+[role="_abstract"]
+The following modifications are explicitly not supported:
+
+* *Creating additional `ServiceMonitor`, `PodMonitor`, and `PrometheusRule` objects in the `openshift-&#42;`, and `kube-&#42;` projects.*
+* *Modifying any resources or objects deployed in the `openshift-monitoring` or `openshift-user-workload-monitoring` projects.* The resources created by the {product-title} monitoring stack are not meant to be used by any other resources, as there are no guarantees about their backward compatibility.
++
+[NOTE]
+====
+The Alertmanager configuration is deployed as a secret resource in the `openshift-monitoring` project. To configure additional routes for Alertmanager, you need to decode, modify, and then encode that secret. This procedure is a supported exception to the preceding statement.
```

Comment:
This is not supported in OSD/ROSA. Any changes will be overwritten and cannot be changed.

Reply:
Done.